src/main/java/com/hivemq/extension/mqtt/message/log/interceptor/ConnectDisconnectEventListener.java

```diff
@@ -78,11 +78,11 @@ public void onConnectionLost(@NotNull final ConnectionLostInput connectionLostIn
 
     @Override
     public void onClientInitiatedDisconnect(@NotNull final ClientInitiatedDisconnectInput clientInitiatedDisconnectInput) {
-        MessageLogUtil.logDisconnect(String.format("Received DISCONNECT from client '%s'.", clientInitiatedDisconnectInput.getClientInformation().getClientId()), clientInitiatedDisconnectInput, verbose);
+        MessageLogUtil.logDisconnect(String.format("Received DISCONNECT from client '%s':", clientInitiatedDisconnectInput.getClientInformation().getClientId()), clientInitiatedDisconnectInput, verbose);
```

Comment:
why ":" there is nothing after that message.

Reply:
There is, in logDisconnect we always append at least the reason, example:
`Received DISCONNECT from client 'clientId': Reason Code: 'NOT_AUTHORIZED'`